frame/balances/src/lib.rs

```diff
@@ -232,8 +238,14 @@ pub mod pallet {
 		/// The means of storing the balances of an account.
 		type AccountStore: StoredMap<Self::AccountId, AccountData<Self::Balance>>;
 
-		/// Weight information for extrinsics in this pallet.
-		type WeightInfo: WeightInfo;
+		/// The ID type for reserves. Use of reserves is deprecated.
```

Comment:
Might be better to mention here what the alternatives are for reserves -- I'm guessing it's both hold and freeze?